### DISCUSSION TITLE
set session limite to 30 minute

### DISCUSSION THREAD

**Discussion by drkamine** [tiny-pilot/tinypilot#1741] [Q&A]
Hello , 
 can  you help me to limit time in session to 30 minute or set a max session login to 1 hours . thanks

**Comment by cghague** [comment]
Hi drkamine, thanks for your question about limiting session length.

I see you've posted this on the discussion board for the community version of TinyPilot. However, the community version of TinyPilot doesn't support user logins; only TinyPilot Pro has that feature. 

Limiting the length of user sessions might be something we can do on TinyPilot Pro, but it could result in unexpected behavior, such as a forced logout while the user is still working.

Could you please provide more context on how you'd expect this to work in practice so I can advise on the best way to proceed? Can you please also confirm if you're using TinyPilot Pro?

**Reply by drkamine** [reply]
Thank you for answering my query. Yes, I use TinyPilot Pro.

**Reply by cghague** [reply]
Thanks for getting back to me!

I need to look into this more with the team. To ensure that I get the appropriate answers, can you please describe your use case in a bit more detail? For example, what would you expect to happen if a user was still working when the session expires?

**Comment by drkamine** [comment]
Thanks sir for helping me with this issue that i have i have a server with
many user système admin connect once a user connect the other session
closed that what i need or any session that opnned for more than 30minute
to be closed. Question 2 if user is connectés i get notification i found
solution to find in log connexion with if and sent email to me is there's
any other easy way. Thanks

**Reply by cghague** [reply]
Thanks for sharing this additional context.

TinyPilot doesn't support sending notifications when a user logs in as an out-of-the-box feature. If you've already found a solution that works for you, it's likely the best option.

I've checked with the team, and unfortunately, it isn't possible to limit user session lengths to 30 minutes or to enforce a logout when another user logs in. I understand this isn't the answer you were hoping for, so as these are interesting ideas, I've logged them as feature requests for future consideration.
